### DISCUSSION TITLE
Do we write out HWP angle quality statistics to a ManifestdB?

### DISCUSSION THREAD

**Discussion by msilvafe** [simonsobs/sotodlib#655] [Q&A] [answered]
In the `g3thwp` angle solution code I see many checks for lower level data quality, dropped packets, etc. and a number of solutions implemented to fix those issues and raise warnings to the logs however I do not currently see the particular samples of interests flagged and recorded in a ManifestdB alongside the angle solutions. Am I missing where this is in the code or is there a plan to add this in the future?

**Comment by ykyohei** [comment, marked as answer]
Yes, there is a plan to add it. It is on the to-do list. Thanks for bringing this up.
https://simonsobs.atlassian.net/wiki/spaces/PRO/pages/235438124/1.06.07.04+HWP

**Comment by yukisakurai** [comment]
It is planed as Kyohei said. We plan to run `site_pipeline/make_hwp_solution.py` as pipeline script for ManifestDB. The main function is `write_solution_h5` in g3thwp.py.
